docs/configuration.md

```diff
@@ -378,9 +402,33 @@ Permitted values are `error`, `warn`, `info` and `debug`. The default is
         disable: false
 
 The storage option is **required** and defines which storage backend is in use.
-You must configure one backend; if you configure more, the registry returns an error.
+You must configure one backend; if you configure more, the registry returns an error. You can choose any of these backend storage drivers:
+
+* The [`filesystem` storage backend](storage-drivers/filesystem.md) uses the local
```

Comment:
I feel like this would be better as a table with name, link to docs, description and caveats.